src/main/java/com/salesforce/dataloader/security/EncryptionUtil.java

```diff
@@ -183,44 +198,35 @@ public String encryptString(String clearText) throws GeneralSecurityException {
     /**
      * Decrypt a string using whatever is the default encryption technique for the system.
      *
-     * @param cipherText
      * @return The decrypted string
-     * @throws NoSuchPaddingException
-     * @throws NoSuchAlgorithmException
-     * @throws BadPaddingException
-     * @throws IllegalBlockSizeException
-     * @throws InvalidKeyException
      */
     public String decryptString(String cipherText) throws GeneralSecurityException {
```

Comment:
Unless supporting migration of existing data, get rid of this function or replace it with a wrapper around its AES-based replacement.

Reply:
Agreed.